### PR TITLE
[transform.math] add bitwise operators

### PR DESCRIPTION
**Pull request by J-N-K**
Tries to solve #227

### CODE REVIEW

File: bundles/org.smarthomej.transform.math/README.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 Transforms the input by applying simple math on it.
 
+The available transformations are 
+
+- `MULTIPLY`: multiply the state by a value
+- `ADD`: add a value to the state
+- `DIVIDE`: dive the state by a value
```
Comment (cweitkamp) [inline bundles/org.smarthomej.transform.math/README.md]:
```suggestion
- `DIVIDE`: divide the state by a value
```

File: bundles/org.smarthomej.transform.math/README.md [hunk 2]
```diff
@@ -11,12 +23,14 @@ Number multiply "Value multiplied by [MULTIPLY(1000):%s]" { channel="<channelUID
 Number add "Value added [ADD(5.1):%s]" { channel="<channelUID>" }
 Number secondsToMinutes "Time [DIVIDE(60):%s]" { channel="<channelUID>" }
 Number subtracted "Value subtracted [ADD(-1):%s]" { channel="<channelUID>" }
+Number bitand "Value OR [BITOR(0x27):%s]" { channel="<channelUID>" }
```
Comment (cweitkamp) [inline bundles/org.smarthomej.transform.math/README.md]:
```suggestion
Number bitor "Value bitor [BITOR(0x27):%s]" { channel="<channelUID>" }
```

File: bundles/org.smarthomej.transform.math/src/main/java/org/smarthomej/transform/math/internal/BitwiseAndTransformationService.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+package org.smarthomej.transform.math.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.transform.TransformationService;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * This {@link TransformationService} adds the given value to the input.
+ *
+ * @author Christoph Weitkamp - Initial contribution
```
Comment (cweitkamp) [inline bundles/org.smarthomej.transform.math/src/main/java/org/smarthomej/transform/math/internal/BitwiseAndTransformationService.java]: Are you sure I am the initial contributor - I am honored? 😉

**Review by cweitkamp — DISMISSED** [review]
Code looks good so far. Some minor comments.

**Comment by J-N-K** [comment]
Thanks. I addressed all your comments.

**Pull request merged**